src/main/java/mate/academy/onlinebookstore/entity/User.java

```diff
@@ -36,4 +46,49 @@ public class User {
 
     @Column(nullable = false)
     private boolean isDeleted = false;
+
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
use LAZY fetch type, it has significant advantages. 
```suggestion
    @ManyToMany(fetch = FetchType.LAZY)
```

Reply:
It's wrong